Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/DriverLicense.hs

```diff
@@ -190,7 +190,7 @@ onVerifyDL verificationReq output = do
   let key = dlCacheKey person.id
   extractedDlAndOperatingCity <- Redis.safeGet key
   case (output.status, verificationReq.issueDateOnDoc, extractedDlAndOperatingCity, verificationReq.driverDateOfBirth) of
-    (Just status, Just issueDate, Just (extractedDL, operatingCity), Just dob) | status == "id_not_found" -> dlNotFoundFallback issueDate (extractedDL, operatingCity) dob verificationReq person
+    (Just status, Just issueDate, Just (extractedDL, operatingCity), Just dob) | status == "id_not_found" -> dlNotFoundFallback issueDate (extractedDL, operatingCity) dob verificationReq person key
     _ -> do
       void $ Redis.del key
```

Comment:
here also we are deleting and you added in other case match as well, why not move it before case match if it has to be deleted always

Reply:
yes we can do that